frontend/playwright.config.ts

```diff
@@ -0,0 +1,56 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+const dotenv = require('dotenv');
+dotenv.config({ path: './.env.test.local' });
+dotenv.config({ path: './.env.test' });
+dotenv.config({ path: './.env' });
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: 'src/__tests__',
+  testMatch: ['/integration/**/*.spec.ts', '/e2e/**/*.spec.ts'],
+  /* Run tests in files in parallel */
+  fullyParallel: !!process.env.E2E,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.E2E ? undefined : 1,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: process.env.E2E ? process.env.E2E_DASHBOARD_URL : 'http://localhost:6006',
```

Comment:
Why this fallback url?

Reply:
6006 is the storybook port. if e2e is used then use whatever url you are testing on (by default it is localhost:4010 but it could be any dashboard url). if you are not doing e2e then it uses storybook server

Reply:
ah gotcha 👍